2025-04-15 10:45:00,688 - DeepSeekDashboard - INFO - 发送 2 条消息到 DeepSeek API...
2025-04-15 10:49:12,913 - DeepSeekDashboard - ERROR - Error processing 2022年中国建筑设计行业的研报.docx: decode() argument 'encoding' must be str, not None
2025-04-15 10:49:12,913 - DeepSeekDashboard - INFO - 发送 2 条消息到 DeepSeek API...
2025-04-15 10:50:15,985 - DeepSeekDashboard - ERROR - Error processing 2022年中国建筑设计行业的研报.docx: decode() argument 'encoding' must be str, not None
2025-04-15 10:50:15,988 - DeepSeekDashboard - INFO - 发送 4 条消息到 DeepSeek API...
2025-04-15 10:50:20,082 - DeepSeekDashboard - INFO - 发送 4 条消息到 DeepSeek API...
2025-04-15 11:00:53,327 - DeepSeekDashboard - INFO - 发送 2 条消息到 DeepSeek API...
2025-04-15 11:10:54,662 - DeepSeekDashboard - ERROR - API 请求失败: Expecting value: line 10 column 1 (char 9)
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\deepseek2.0\.venv\Lib\site-packages\requests\models.py", line 974, in json
    return complexjson.loads(self.text, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python312\Lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 10 column 1 (char 9)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\deepseek2.0\deepseek-ai.py", line 105, in query_deepseek
    response_data = response.json()
                    ^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\PycharmProjects\deepseek2.0\.venv\Lib\site-packages\requests\models.py", line 978, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 10 column 1 (char 9)
2025-04-15 11:12:20,185 - DeepSeekDashboard - INFO - 发送 2 条消息到 DeepSeek API...
2025-04-15 11:12:50,356 - DeepSeekDashboard - INFO - 发送 2 条消息到 DeepSeek API...
2025-04-15 11:15:59,095 - DeepSeekDashboard - INFO - 发送 2 条消息到 DeepSeek API...
2025-04-15 11:16:19,996 - DeepSeekDashboard - INFO - 发送 4 条消息到 DeepSeek API...
2025-04-15 11:36:30,554 - DeepSeekDashboard - INFO - 发送 2 条消息到 DeepSeek API...
2025-04-15 12:07:31,662 - git.cmd - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2025-04-15 12:07:31,666 - git.cmd - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2025-04-15 14:03:06,049 - DeepSeekDashboard - INFO - 发送 2 条消息到 DeepSeek API...
